scylla/src/cluster/state.rs

```diff
@@ -185,11 +191,14 @@ impl ClusterState {
         }
     }
 
-    /// Access keyspaces details collected by the driver
-    /// Driver collects various schema details like tables, partitioners, columns, types.
-    /// They can be read using this method
-    pub fn get_keyspace_info(&self) -> &HashMap<String, Keyspace> {
-        &self.keyspaces
+    /// Access keyspace details collected by the driver.
+    pub fn get_keyspace(&self, keyspace: impl AsRef<str>) -> Option<&Keyspace> {
+        self.keyspaces.get(keyspace.as_ref())
+    }
+
+    /// Returns an iterator over keyspaces.
+    pub fn get_keyspaces_iter(&self) -> impl Iterator<Item = (&String, &Keyspace)> {
+        self.keyspaces.iter()
     }
```

Comment:
Nit: This name sounds like there is already some iterator in ClusterState, and we are retrieving it.
I'd change this to `keyspaces_iter()`.